doc/getting_started/getting_started_go.rst

```diff
@@ -245,3 +245,207 @@ To send bare Lua code for execution, use ``Eval``:
+        -   README with examples, code comments
+        -   README with examples
+
+    *   -   Testing / CI / CD
+        -   GitHub Actions
+        -   Travis CI
+        -   GitHub Actions
+
+    *   -   Github Stars
+        -   127
+        -   43
+        -   12
+
+    *   -   Static Analysis
+        -   No
+        -   golint
+        -   golangci-lint
+
+    *   -   Packaging
+        -   go get
+        -   go get
+        -   go get
+
+    *   -   Code coverage
+        -   No
+        -   No
+        -   No
+
+    *   -   msgpack driver
+        -   `vmihailenco/msgpack/v2 <https://github.com/vmihailenco/msgpack/tree/v2>`_
+        -   `tinylib/msgp <https://github.com/tinylib/msgp>`_
+        -   `vmihailenco/msgpack/v5 <https://github.com/vmihailenco/msgpack/tree/v5>`_
+
+    *   -   Async work
+        -   Yes (channels)
+        -   Yes (channels)
+        -   Yes (channels)
+
+    *   -   Batching
+        -   No
+        -   No
+        -   No
+
+    *   -   Schema reload
+        -   Yes (manual pull)
+        -   Yes (manual pull)
+        -   Yes (manual pull)
+
+    *   -   Space / index names
+        -   Yes
+        -   Yes
+        -   Yes
+
+    *   -   Tuples as structures
+        -   Yes (structure and marshall functions must be predefined in Go code)
+        -   No
+        -   Yes (structure and marshall functions must be predefined in Go code)
+
+    *   -   Access tuple fields by names
+        -   Only if marshalled to structure
+        -   No
+        -   Only if marshalled to structure
+
+    *   -   `SQL <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`_ support
+        -   No (`#62 <https://github.com/tarantool/go-tarantool/issues/62>`_), eval supported
+        -   No (`#18 <https://github.com/viciious/go-tarantool/issues/18>`_, closed), eval supported
+        -   No, eval supported
+
+    *   -   `Interactive transactions <https://www.tarantool.io/en/doc/latest/book/box/stream/>`_
+        -   No (`#101 <https://github.com/tarantool/go-tarantool/issues/101>`_)
+        -   No
+        -   No
+
+    *   -   `Varbinary <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes (with in-built language tools)
+        -   Yes (with in-built language tools)
+        -   Questionable (decoding is weird with in-built language tools)
+
+    *   -   `UUID <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes
+        -   No
+        -   No
+
+    *   -   Decimal support
+        -   No (`#96 <https://github.com/tarantool/go-tarantool/issues/96>`_)
+        -   No
+        -   No
+
+    *   -   `EXT_ERROR <https://www.tarantool.io/ru/doc/latest/dev_guide/internals/msgpack_extensions/#the-error-type>`_
+            support
+        -   No
+        -   No
+        -   No
+
+    *   -   `[Soon] Datetime <https://github.com/tarantool/tarantool/discussions/6244>`_ support
```

Comment:
I would strip 'Soon' from the website page.

Reply:
Removed all "soon"s